### DISCUSSION TITLE
Printing only the annotations of a PDF file

### DISCUSSION THREAD

**Discussion by avikla** [sumatrapdfreader/sumatrapdf#3422] [Ideas]
It would be great to just be able to print the annotations that are marked in a PDF file.

**Comment by GitHubRulesOK** [comment]
generally the annotations are stored in the tail of the file and simply IN EFFECT float over the pages so it would be like printing a sheet of tracing  paper you can do that with Acrobat or other editors (here from my other favourite) but it would be pointless use of resources in SumatraPDF, Annotations are NOT attached to the objects used for highlighting simply a spatial co-ordinate that may be assigned to any file or page.

![image](https://user-images.githubusercontent.com/12754913/230405533-59976bb7-a785-43da-8a6e-77337056846d.png)
